Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG PY_VERSION=3.7
+FROM continuumio/miniconda3:4.8.2-alpine AS builder
+
+EXPOSE 8888
+
+LABEL maintainer.name="mosdef-hub"\
+      maintainer.url="https://mosdef.org"
+
+ENV PATH /opt/conda/bin:$PATH
+
+USER root
+
+ADD . /mbuild
+
+WORKDIR /mbuild
+
+RUN conda update conda -yq && \
+	conda config --set always_yes yes --set changeps1 no && \
+	conda config --add channels omnia && \
+	conda config --add channels conda-forge && \
+	conda config --add channels mosdef && \
+	conda create -n mbuild-docker python=$PY_VERSION && \
+	. /opt/conda/etc/profile.d/conda.sh && \
+	conda activate mbuild-docker && \
+	conda install python=$PY_VERSION nomkl --file requirements-dev.txt && \
+        python setup.py install && \
+	echo "conda activate mbuild-docker" >> \
+	/home/anaconda/.profile && \
+	conda clean -afy && \
+	chown -R anaconda:anaconda /mbuild && \
+	chown -R anaconda:anaconda /opt && \
+	chown -R anaconda:anaconda /home/anaconda
+
+USER anaconda
+
+WORKDIR /scratch
+
+CMD [ "/bin/sh", "--login -i" ]
```

Comment:
Note that i still had to run 
`. /opt/conda/etc/profile.d/conda.sh` when in the shell to be able to run `conda activate`